### ISSUE TITLE
Suggest new issues

### ISSUE THREAD

**Issue by ananya-agrawal** [opencodeiiita/Opencode-Collaborative-19#61]

**Comment by aswinzz** [comment]
Do not create issues directly.. comment the issues here and we will be creating issues based on your comments.

**Comment by zerefwayne** [comment]
### 10 points for each issue accepted by us.

Make sure the issues you comment here are meaningful and aren't duplicate.

**Comment by NamanDeept** [comment]
Instead of keeping the participants section ,mentors section and the projects section in a single page one after the other, we can make 3 tab buttons of participants ,mentors and the projects. So that when the user clicks participants ..the participants section will be shown and other respectively.This will reduce the page size

**Comment by zerefwayne** [comment]
> Instead of keeping the participants section ,mentors section and the projects section in a single page one after the other, we can make 3 tab buttons of participants ,mentors and the projects. So that when the user clicks participants ..the participants section will be shown and other respectively.This will reduce the page size

**Comment by mohammed-adil** [comment]
Add pagination to participant section, since many participants are sending new PR to update participant count with their details.

**Comment by punisher1151** [comment]
Fix : Add scroll back to top button in footer

**Comment by ashish4321** [comment]
@ananya-agrawal 
Move the Copyright content from middle of the footer to bottom left. It does not look nice in the middle.
Also remove the extra space between footer and projects section.

**Comment by ashish4321** [comment]
@ananya-agrawal 
Fix: Clicking on the Partcipants and Mentors card should redirect to their respective Github profiles.

**Comment by mohammed-adil** [comment]
@zerefwayne @ananya-agrawal  Lazy-load all the images for smooth user experience.

**Comment by B3ingALearner** [comment]
@Anupam-dagar  Logo of opencode is not visible due to colour of Nav-Bar

**Comment by zerefwayne** [comment]
@B3ingALearner Accepted. :+1: You may open this issue.

**Comment by ashish4321** [comment]
@ananya-agrawal @zerefwayne 
Add animation to OpenCode Logo and display only the animated logo when the page is loading

**Comment by KeshavSharma21099** [comment]
@ananya-agrawal 
Add OpenCode'19 logo to the title bar of the OpenCode'19 Tab.

**Comment by sourabh1031** [comment]
All the participants and mentors should be displayed in single line with arrow on left and right side to scroll and automatic scrolling should also be done

**Comment by zerefwayne** [comment]
@KeshavSharma21099 Issue accepted! It is called a **favicon** of the webpage.

**Comment by zerefwayne** [comment]
@sourabh1031 We have already opened an issue to implement pagination for the same.

**Comment by mohammed-adil** [comment]
@zerefwayne @ananya-agrawal Adding smooth scrolling throughout page that includes clicking the links from navbar to footerr of the page.

**Comment by vishnuk7** [comment]
change the font

**Comment by sourabh1031** [comment]
github zulip integration

**Comment by sourabh1031** [comment]
gradient is too dark to view the text "Welcome ....". header color is also looking odd with gradient

**Comment by punisher1151** [comment]
facebook animated bar could be added as in previous site
![screenshot from 2019-01-15 18-00-50](https://user-images.githubusercontent.com/42278252/51180870-9d7d5d80-18ef-11e9-8c13-6c508f7203ce.png)

**Comment by punisher1151** [comment]
Color of fb ,twitter and github icon should be white in footer

**Comment by sourabh1031** [comment]
Participants cards are repeating

**Comment by helios1101** [comment]
![screenshot from 2019-01-18 19-04-34](https://user-images.githubusercontent.com/26597930/51389973-f2b9a900-1b53-11e9-9efb-6c19ae4fb282.png)
Upon clicking "get started" button, which is taking people directly to participants section, instead a smooth scroll effect must be there which must slide to participants section.

**Comment by ananya-agrawal** [comment]
@helios1101 we have an issue already opened to implement smooth scrolling throughout the website

**Comment by Rattlecode17sh** [comment]
![screenshot 44](https://user-images.githubusercontent.com/38866385/51404978-00cef000-1b7b-11e9-93a2-cd6b8745c5d3.png)
![screenshot 45](https://user-images.githubusercontent.com/38866385/51404979-00cef000-1b7b-11e9-9dab-6b33398f5df0.png)

When viewed in mobile view participant cards looks very dull even College is diaplayed outside Cards in some cases. We can fix this.

**Comment by Rattlecode17sh** [comment]
We can add a timer to show how much time is left for opencode'19 to end

**Comment by sourabh1031** [comment]
flipping card effect on participants causes motto line of participants to display on image.

**Comment by prijeshb** [comment]
> flipping card effect on participants causes motto line of participants to display on image. also names of the participants on card flipping gets reversed.

@sourabh1031  not all i guess only of mentors

**Comment by sourabh1031** [comment]
![screenshot from 2019-01-22 11-44-07](https://user-images.githubusercontent.com/25590627/51515804-1a9e5a80-1e3b-11e9-8877-ecf81717f3d0.png)
Website broke

**Comment by devgupta2607** [comment]
Navbar links can have hover effects and navbar change color as it reaches the sections like participant, mentor and projects and colors which suit the logo

**Comment by punisher1151** [comment]
Flipping cards of some users doesn't have any quote or text . A 
![screenshot from 2019-01-25 22-14-13](https://user-images.githubusercontent.com/42278252/51759723-e0f57a00-20ee-11e9-9b1b-0fe9e34b2580.png)
default one could be provided

**Comment by ananya-agrawal** [comment]
> github zulip integration

@sourabh1031 github zulip integration is quite easy to do but for that you need to have organisation rights but sorry we can't give you that :stuck_out_tongue:

**Comment by ananya-agrawal** [comment]
> Navbar links can have hover effects and navbar change color as it reaches the sections like participant, mentor and projects and colors which suit the logo

@devgupta2607 you can open an issue to have hover effect on navbar links

**Comment by ananya-agrawal** [comment]
> Flipping cards of some users doesn't have any quote or text . A default one could be provided ,as here of the middle one

@punisher1151 what sort of default text you are thinking to put?

**Comment by sourabh1031** [comment]
> > github zulip integration
> 
> @sourabh1031 github zulip integration is quite easy to do but for that you need to have organisation rights but sorry we can't give you that stuck_out_tongue

@ananya-agrawal   then  we can try github slack integration i think that does not require admin rights :P

**Comment by ananya-agrawal** [comment]
> > > github zulip integration
> > 
> > 
> > @sourabh1031 github zulip integration is quite easy to do but for that you need to have organisation rights but sorry we can't give you that stuck_out_tongue
> 
> @ananya-agrawal then we can try github slack integration i think that does not require admin rights :P

@sourabh1031 but we have moved to zulip chat officially :stuck_out_tongue:

**Comment by ananya-agrawal** [comment]
> When viewed in mobile view participant cards looks very dull even College is diaplayed outside Cards in some cases. We can fix this.
@Rattlecode17sh yeah, you can open issue to fix cards

**Comment by anupm12** [comment]
An issue with night mode and pagination -
1. Switch to night mode
2. Then navigate to any page using pagination

The new cards will be in normal mode, 
After this turn off night mode and the cards in normal mode will become dark.

**Comment by anupm12** [comment]
Images of few participants are in the participant images folder but they are not being displayed.
My image too 😅

**Comment by sourabh1031** [comment]
add 'fork me on github' css ribbon .

**Comment by anupm12** [comment]
The present spinner looks bad.
It can logo of the website or some other animation which doesn't use bright orange color at least 😅

**Comment by punisher1151** [comment]
> > Flipping cards of some users doesn't have any quote or text . A default one could be provided ,as here of the middle one
> 
> @punisher1151 what sort of default text you are thinking to put?
like "Web Developer" or "Talk is cheap. Show me the code" etc.

**Comment by deepakDOTexe** [comment]
@ananya-agrawal @zerefwayne The Nav-bar gets hidden when we scroll down the page. We can fix it's position at the top even when a user scrolls down the page.

**Comment by sourabh1031** [comment]
Participants card images and of mentors should be circle

**Comment by sourabh1031** [comment]
Fork and star favicon on website should work as an active link pointing to github repo of project

**Comment by ananya-agrawal** [comment]
:+1: do open these issues at the earliest

**Comment by anupm12** [comment]
> As the number of participants is now increased even though the images were earlier compressed has to be compressed again(less than 100kb per image) to decrease the overall size of the website and also the load time
> Also, because of this the pagination works very slow.

@ananya-agrawal you have opened this issue.....will I get any points for this? 😅

**Comment by Rattlecode17sh** [comment]
![screenshot 58](https://user-images.githubusercontent.com/38866385/51853377-25cb1c00-234e-11e9-834d-c3c365ed9a35.png)

@ananya-agrawal i think after compressing images of participants. Ratio between height and width of cards is unequal. We can fix this

**Comment by Rattlecode17sh** [comment]
@ananya-agrawal we can also make a leader board in the collaborative website same as in previous editions of open code showing all the contributions and points alloted to participants

**Comment by ananya-agrawal** [comment]
> > As the number of participants is now increased even though the images were earlier compressed has to be compressed again(less than 100kb per image) to decrease the overall size of the website and also the load time
> > Also, because of this the pagination works very slow.
> 
> @ananya-agrawal you have opened this issue.....will I get any points for this?

Yeah you will get points :smile:

**Comment by anupm12** [comment]
@ananya-agrawal Issue with active pagination page 

![image](https://user-images.githubusercontent.com/35293695/51861408-21a8f980-2362-11e9-97b7-841a4436fac2.png)

**Comment by ananya-agrawal** [comment]
@anupm12 is navbar coming on scroll? I am not quite sure about that, let me know once

**Comment by anupm12** [comment]
@ananya-agrawal Navbar is fixed on scroll.

**Comment by ananya-agrawal** [comment]
@anupm12 check it once as for me it is not fixed, https://opencodeiiita.github.io/Opencode-Collaborative-19/

**Comment by anupm12** [comment]
@ananya-agrawal Yes, It's not fixed, but I was the one who did that and even now on my fork it is fixed

![image](https://user-images.githubusercontent.com/35293695/51864744-92541400-236a-11e9-9f48-913ff5ff1211.png)

**Comment by anupm12** [comment]
It should be working fine as my code is still there

![image](https://user-images.githubusercontent.com/35293695/51864879-f37be780-236a-11e9-8a33-fda6f870f9ab.png)

![image](https://user-images.githubusercontent.com/35293695/51864972-363dbf80-236b-11e9-91a7-b50b6e6ee919.png)

**Comment by palak2001** [comment]
Issue Suggestion:
Delete the particularly uploaded(useless) file.

**Comment by Rattlecode17sh** [comment]
![screenshot 60](https://user-images.githubusercontent.com/38866385/51871102-71e08580-237b-11e9-8273-1703f81f78eb.png)


@ananya-agrawal  i think aditya kumar filled the JSON file twice that's showing 2 cards of him. We can remove one card

**Comment by Rattlecode17sh** [comment]
> > @ananya-agrawal we can also make a leader board in the collaborative website same as in previous editions of open code showing all the contributions and points alloted to participants
> 
> leader board is already made by @Jigar3 and will be up by tomorrow 😛

okay cool

**Comment by sourabh1031** [comment]
Mentors cards are bit rectangular. fix them as well

**Comment by sourabh1031** [comment]
position of the header should be fixed in website

**Comment by Rattlecode17sh** [comment]
![screenshot_2019-01-29-04-23-40-403 1](https://user-images.githubusercontent.com/38866385/51896985-a8e28580-23d3-11e9-9fa2-6c565ded3788.jpeg)
![screenshot_2019-01-29-04-23-34-204](https://user-images.githubusercontent.com/38866385/51896987-a97b1c00-23d3-11e9-9791-414bc769476d.jpeg)

Same image for two different participants. We can fix this

**Comment by Rattlecode17sh** [comment]
![screenshot_2019-01-29-04-24-25-622](https://user-images.githubusercontent.com/38866385/51897048-d2031600-23d3-11e9-89b4-a1874da1bd75.jpeg)

Image size is not in same ratio in mobile view. It is little bit shrinked

**Comment by anupm12** [comment]
@ananya-agrawal Fork me on GitHub ribbon covers hamburger menu in mobile.

![image](https://user-images.githubusercontent.com/35293695/52003993-1550a800-24ec-11e9-830c-ece559c2064b.png)

**Comment by anupm12** [comment]
Make logo and name clickable

![image](https://user-images.githubusercontent.com/35293695/52004232-a6278380-24ec-11e9-9949-1d9e15d91a66.png)

**Comment by thenullterminator** [comment]
@ananya-agrawal The top buttons in leader board are not visible in mobile view

**Comment by ananya-agrawal** [comment]
> @ananya-agrawal The top buttons in leader board are not visible in mobile view

Yeah open issue in leaderboard's repo

**Comment by ananya-agrawal** [comment]
> @ananya-agrawal Fork me on GitHub ribbon covers hamburger menu in mobile.
> 
> ![image](https://user-images.githubusercontent.com/35293695/52003993-1550a800-24ec-11e9-830c-ece559c2064b.png)

How are you planning to solve this ?

**Comment by anupm12** [comment]
> > @ananya-agrawal Fork me on GitHub ribbon covers hamburger menu in mobile.
> > ![image](https://user-images.githubusercontent.com/35293695/52003993-1550a800-24ec-11e9-830c-ece559c2064b.png)
> 
> How are you planning to solve this ?

For a mobile screen, this ribbon can be moved to left or we can have a slider kind of thing below the hamburger menu with GitHub logo and hovering on which the slider would slide from right to left with the fork me message.

**Comment by anupm12** [comment]
![image](https://user-images.githubusercontent.com/35293695/52073617-f3b7f500-25ad-11e9-8a30-4096d01156d1.png)

Those border between icons

**Comment by anupm12** [comment]
Pagination active content comes over navbar

![image](https://user-images.githubusercontent.com/35293695/52075252-d71dbc00-25b1-11e9-983b-e03ec68a2af9.png)

**Comment by sourabh1031** [comment]
Delete duplicate participants details

**Comment by ananya-agrawal** [comment]
> ![image](https://user-images.githubusercontent.com/35293695/52073617-f3b7f500-25ad-11e9-8a30-4096d01156d1.png)
> 
> Those border between icons

@anupm12 this is not happening for me, are you sure this is actually happening in the main code. Maybe this is something happening in your fork

**Comment by sourabh1031** [comment]
fork and star favicon is not visible 
![screenshot from 2019-02-02 17-47-09](https://user-images.githubusercontent.com/25590627/52164147-d81a2f00-2712-11e9-9bf9-acc8043b85ff.png)

**Comment by ananya-agrawal** [comment]
> fork and star favicon is not visible
> ![screenshot from 2019-02-02 17-47-09](https://user-images.githubusercontent.com/25590627/52164147-d81a2f00-2712-11e9-9bf9-acc8043b85ff.png)

They are coming for me, check it again once

**Comment by anupm12** [comment]
The color behind the logo is different than the background color of the navbar.

![image](https://user-images.githubusercontent.com/35293695/52167553-a0c37680-2742-11e9-9fd9-9d1365b9d585.png)

**Comment by anupm12** [comment]
Textfield in the card has a fixed height

![image](https://user-images.githubusercontent.com/35293695/52167567-eed87a00-2742-11e9-8446-da56e2efaa07.png)

**Comment by sourabh1031** [comment]
Hamburger menu is not visible in mobile

**Comment by sourabh1031** [comment]
disable horizontal overflow in mobile view

**Comment by Rattlecode17sh** [comment]
Can we add the Personal website section too??

**Comment by Rattlecode17sh** [comment]
@ananya-agrawal  we can add forks and stars on the projects cards itself fro each and every project so that we can get additional information about project from collaborative website only

**Comment by Rattlecode17sh** [comment]
We can add a custom scroll bar @ananya-agrawal

**Comment by Rattlecode17sh** [comment]
We can add a search bar and when we enter name of any project, mentor or participant it should take us to the card of those mentor, participant, project. So that we can directlt get to that card instead of going through pagination

**Comment by Rattlecode17sh** [comment]
![screenshot 69](https://user-images.githubusercontent.com/38866385/52210531-56aad400-28ad-11e9-8c40-7a9a7641ac3a.png)

Well that facebook effect on right side looks cool in previous edition we can add this in collaborative website this year too

**Comment by Rattlecode17sh** [comment]
![screenshot 71](https://user-images.githubusercontent.com/38866385/52210779-fec09d00-28ad-11e9-84dd-0368eb86d1cc.png)
In some participant cards images are not uniform. I mean images are not completely overlapped in mobile. We can fix this @ananya-agrawal

**Comment by Rattlecode17sh** [comment]
Lots of 404 errors in js console. We can fix this

**Comment by Rattlecode17sh** [comment]
@ananya-agrawal we can hide the navbar in landing page and when we scroll down we can make it appear as it looks really cool after that.

**Comment by Rattlecode17sh** [comment]
Right now preloader looks very simple we can make it more attractive

**Comment by Rattlecode17sh** [comment]
We can add svg animations to opencode logo so that it looks cool @ananya-agrawal

**Comment by ananya-agrawal** [comment]
> Can we add the Personal website section too @ananya-agrawal ??
 
That is already done

**Comment by ananya-agrawal** [comment]
> @ananya-agrawal we can hide the navbar in landing page and when we scroll down we can make it appear as it looks really cool after that.

https://opencodeiiita.github.io/CodeFemme/ yeah we can do something like done here

**Comment by ananya-agrawal** [comment]
> We can add svg animations to opencode logo so that it looks cool @ananya-agrawal

Add where ?

**Comment by Rattlecode17sh** [comment]
> > Can we add the Personal website section too @ananya-agrawal ??
> 
> That is already done

I mean in collaborative website

**Comment by ananya-agrawal** [comment]
> ![screenshot 69](https://user-images.githubusercontent.com/38866385/52210531-56aad400-28ad-11e9-8c40-7a9a7641ac3a.png)
> 
> Well that facebook effect on right side looks cool in previous edition we can add this in collaborative website this year too

I guess someone suggested this issue some days back do check once if it was approved or not above

**Comment by ananya-agrawal** [comment]
> > > Can we add the Personal website section too @ananya-agrawal ??
> > 
> > 
> > That is already done
> 
> I mean in collaborative website

Yeah there was an issue created to add personal website link in the participant cards itself, though not sure if it is merged or not

**Comment by Rattlecode17sh** [comment]
> We can add svg animations to opencode logo so that it looks cool @ananya-agrawal



> > We can add svg animations to opencode logo so that it looks cool @ananya-agrawal
> 
> Add where ?

I mean when website loads instead of that preloader right now and we can use opencode logo

**Comment by Rattlecode17sh** [comment]
> > > > Can we add the Personal website section too @ananya-agrawal ??
> > > 
> > > 
> > > That is already done
> > 
> > 
> > I mean in collaborative website
> 
> Yeah there was an issue created to add personal website link in the participant cards itself, though not sure if it is merged or not

Okay i never saw it that's why i suggested this issue

**Comment by Rattlecode17sh** [comment]
> > ![screenshot 69](https://user-images.githubusercontent.com/38866385/52210531-56aad400-28ad-11e9-8c40-7a9a7641ac3a.png)
> > Well that facebook effect on right side looks cool in previous edition we can add this in collaborative website this year too
> 
> I guess someone suggested this issue some days back do check once if it was approved or not above

Yeah you are correct it was previously suggested by @punisher1151 .

**Comment by sourabh1031** [comment]
in pagination of participants and projects area when one goes from one active page to another , after clicking the screen becomes  unscrollable with arrow key. only touch pad works

**Comment by anupm12** [comment]
@ananya-agrawal 
Problem in sorting
![image](https://user-images.githubusercontent.com/35293695/52291247-f3e23700-2997-11e9-844b-3cca1ec4b622.png)

**Comment by sourabh1031** [comment]
in mobile view github fork icon hides behind cards and other div it should be above
![screenshot from 2019-02-06 10-37-09](https://user-images.githubusercontent.com/25590627/52321575-70f2c800-29fb-11e9-8d80-b41f97d08d6f.png)

**Comment by prijeshb** [comment]
@ananya-agrawal  Should give first priority to this
Many request are failed and redudant user profile
![screenshot 161](https://user-images.githubusercontent.com/16492616/52333209-a8c13600-2a22-11e9-9f9e-f4fde22c5f5b.png)
I think so typo https instead of http
![screenshot 163](https://user-images.githubusercontent.com/16492616/52333387-24bb7e00-2a23-11e9-90f0-80a5a9c49373.png)

**Comment by anupm12** [comment]
> > @ananya-agrawal
> > Problem in sorting
> > ![image](https://user-images.githubusercontent.com/35293695/52291247-f3e23700-2997-11e9-844b-3cca1ec4b622.png)
> 
> I can see there was a pr for sorting #237 and the same problem is mentioned in the pr also
> @sourabh1031 @pradeepgangwar was the problem solved or the pr was merged without it getting fixed?

@ananya-agrawal 
I think it is not solved.

**Comment by anupm12** [comment]
@ananya-agrawal Duplicate data in participant.json

![image](https://user-images.githubusercontent.com/35293695/52365816-5574d500-2a6e-11e9-9ed6-ab3849a0a11c.png)

**Comment by sourabh1031** [comment]
> @ananya-agrawal Duplicate data in participant.json
> 
> ![image](https://user-images.githubusercontent.com/35293695/52365816-5574d500-2a6e-11e9-9ed6-ab3849a0a11c.png)

@anupm12 i have already mentioned this issue above but not accepted by @ananya-agrawal

**Comment by sourabh1031** [comment]
> > > @ananya-agrawal
> > > Problem in sorting
> > > ![image](https://user-images.githubusercontent.com/35293695/52291247-f3e23700-2997-11e9-844b-3cca1ec4b622.png)
> > 
> > 
> > I can see there was a pr for sorting #237 and the same problem is mentioned in the pr also
> > @sourabh1031 @pradeepgangwar was the problem solved or the pr was merged without it getting fixed?
> 
> @ananya-agrawal
> I think it is not solved.

yes @anupm12 @ananya-agrawal the problem was not fixed at that time , check the code once the same logic sorts mentors properly but in participants creates a glitch

**Comment by Rattlecode17sh** [comment]
![screenshot 90](https://user-images.githubusercontent.com/38866385/52399865-7f1c1380-2ae3-11e9-98d6-855e20d036fc.png)
@ananya-agrawal  Wall street project is mentioned twice in issues section. Fix this

**Comment by Rattlecode17sh** [comment]
![screenshot 91](https://user-images.githubusercontent.com/38866385/52400177-6829f100-2ae4-11e9-92f4-6478a4324b66.png)
We can add hover effect on different sections in navbar such as project, mentor and participant to make it more attractive

**Comment by kesari007** [comment]
After clicking **Participants** in the nav bar once, if we click **Mentor** then it doesnt take us to the mentor section, rather scrolls down just a bit and remains in the participant section.
Same is with **Mentors** and **Projects** when clicked one after other.
@ananya-agrawal

**Comment by ananya-agrawal** [comment]
> ![screenshot 91](https://user-images.githubusercontent.com/38866385/52400177-6829f100-2ae4-11e9-92f4-6478a4324b66.png)
> We can add hover effect on different sections in navbar such as project, mentor and participant to make it more attractive

That was done some days back, I don't remember by whom but check it once before doing this

**Comment by Rattlecode17sh** [comment]
> > ![screenshot 91](https://user-images.githubusercontent.com/38866385/52400177-6829f100-2ae4-11e9-92f4-6478a4324b66.png)
> > We can add hover effect on different sections in navbar such as project, mentor and participant to make it more attractive
> 
> That was done some days back, I don't remember by whom but check it once before doing this

Okay i will check it

**Comment by kesari007** [comment]
The **arrow** mouse cursor does not change to **Hand** cursor when we hover through **Trailer** in Navbar.

**Comment by kesari007** [comment]
The mouse cursor starts to Flicker when placed on respective page in the participant section.
For example if we are at page 2 of participant section and i place the cursor on 2,then the mouse pointer starts to flicker.

**Comment by kesari007** [comment]
College name of participant going out of card.

![screenshot from 2019-02-11 19-01-51](https://user-images.githubusercontent.com/25413808/52566358-96366a80-2e2f-11e9-949b-e17ae1d4e69a.png)
